packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -498,22 +498,33 @@ export function makeWallet({
       },
     );
 
+    // Try reclaiming any of our payments that were left unclaimed.
+    const claimMyLeftovers = () =>
+      keywords.forEach((keyword, i) => {
+        const purseOrUndefined = purseKeywordRecord[keyword];
+        // eslint-disable-next-line no-use-before-define
+        addPayment(paymentPs[i], purseOrUndefined);
```

Comment:
What happens if `addPayment` errors? That would return a rejected promise, which `claimMyLeftovers` just drops on the floor, right? So failures in depositing payments are entirely independent and do not cause `claimMyLeftovers` to throw?

Reply:
I believe `purseKeywordRecord[keyword]` is always defined here, since the purseKeywordRecord was created through iterating through the keywords.

Reply:
Thanks for these comments.  I've cleared it up.